scripts/webpack/webpack.common.js

```diff
@@ -108,6 +136,11 @@ module.exports = {
           },
         ],
       },
+      {
+        test: /\.css$/,
+        // include: MONACO_DIR, // https://github.com/react-monaco-editor/react-monaco-editor
```

Comment:
The docs suggest we should limit CSS to only the files under:
```
const MONACO_DIR = path.resolve(__dirname, './node_modules/monaco-editor');
```
but that does not seem to be necessary since most css is managed by the scss plugin